dev/ci/user-overlays/12653-SkySkimmer-cumul-syntax.sh

```diff
@@ -0,0 +1,15 @@
+if [ "$CI_PULL_REQUEST" = "12653" ] || [ "$CI_BRANCH" = "cumul-syntax" ]; then
```

Comment:
Don't you need to use the current PR number / branch if you want to test the command?

Reply:
curses